orc8r/cloud/deploy/terraform/orc8r-helm-aws/templates/orc8r-values.tpl

```diff
@@ -54,6 +54,8 @@ controller:
   image:
     repository: ${docker_registry}/controller
     tag: "${docker_tag}"
+    env:
+      version_tag: ${docker_tag}
```

Comment:
Here and below: quote templated values, since idk what the yaml rules are for what constitutes a string and we shouldn't have to think about what values the templated value could take
```suggestion
      version_tag: "${docker_tag}"
```